eng/pipelines/runtime.yml

```diff
@@ -1753,7 +1753,7 @@ extends:
     - stage: SourceBuild
       displayName: Source Build Validation
       dependsOn: []
-      condition: eq(variables['isRollingBuild'], true)
+      # condition: eq(variables['isRollingBuild'], true)
```

Comment:
```suggestion
      condition: eq(variables['isRollingBuild'], true)
```

Reply:
@jkoritzinsky, please apply this before merging. The source build job is running here https://github.com/dotnet/runtime/pull/115272/checks?check_run_id=41598204946